source/uwp/Renderer/lib/AdaptiveChoiceSetInputRenderer.cpp

```diff
@@ -169,6 +169,28 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
                     // Otherwise, leave all options unset
                     radioButton.IsChecked(IsChoiceSelected(values, input));
                 }
+
+                radioButton.PreviewKeyDown(
+                    [this, stackPanel](winrt::IInspectable const& sender, winrt::KeyRoutedEventArgs const& args) -> void
+                    {
+                        std::uint32_t currentIndex;
+                        auto children = stackPanel.Children();
+                        auto size = children.Size();
+                        if (const auto isButtonFound = children.IndexOf(sender.as<winrt::RadioButton>(), currentIndex))
+                        {
+                            if (args.Key() == winrt::VirtualKey::Down)
+                            {
+                                currentIndex = (currentIndex + 1) % size;
+                                children.GetAt(currentIndex).as<winrt::RadioButton>().IsChecked(true);
```

Comment:
move this line outside of the if/else to avoid writing it twice:)

Reply:
I can restructure this if you prefer (add a top level conditional that checks for both arrow keys)
Moving it outside the current if/else created issues when using the Tab key

Reply:
you're right, didn't realize :)